go/textutil/strings.go

```diff
@@ -90,6 +90,19 @@ func SingleWordCamel(w string) string {
 	return strings.ToUpper(w[0:1]) + strings.ToLower(w[1:])
 }
 
+var multiWordSplitterRegexp = regexp.MustCompile(`[-_.\s]+`)
+
+// SingleWordCamel takes a single word and returns is in Camel case; basically
+// just capitalizing the first letter and making sure the rest are lower case.
+func MultiWordCamel(w string) string {
```

Comment:
This looks like PascalCase to me. No? The test seems to confirm it. Obviously very minor, but would be better to use an existing well defined term IMO.

Reply:
That's right! Changed terminology to `PascalCase`. Turbo Pascal 3 forever.